administrator/components/com_guidedtours/src/Controller/StepController.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Don't be lazy when copy pasting code from elsewhere - comments are as important as code